docs/user_manual.md

```diff
@@ -23,6 +24,17 @@ Note that this guide assumes that you are familiar with [Docker](http://docker.i
   * [Using Deployment Groups](#using-deployment-groups)
 * [Once Inside The Container](#once-inside-the-container)
 
+
+
+Source Code Setup
+---
+(For Helios Developers and Contributors)
+1) Under the parent level helios source directory run "mvn clean install." This will run the integration tests, which may take some time. To avoid running the tests, add the "-DskipTests" option. 
+2) Note that the above build command will produce a generate directory under "helios-client/target/generated-sources/templated" which may need to be added as a source direcory to your IDE manually.
+3) In order to run the Helios integration tests, first run: mvn -P build-images -P build-solo package -DskipTests=true -Dmaven.javadoc.skip=true -B -V -pl helios-services
+4) Note that in order to submit pull requests to Github, your account must be properly setup.  See: https://confluence.spotify.net/pages/viewpage.action?pageId=10683655.
```

Comment:
Thanks - I'll remove (4) - wondering if there is a better place for that?  Perhaps, the Onboarding checklist?